components/project-card/project-card-simple.jsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import ReactGA from 'react-ga';
+import { Link } from 'react-router';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import ModerationPanel from '../moderation-panel.jsx';
+import Creators from './meta/creators.jsx';
+import Thumbnail from './meta/thumbnail.jsx';
+import Title from './meta/title.jsx';
+import Description from './meta/description.jsx';
+import GetInvolved from './meta/get-involved.jsx';
+import WhyInteresting from './meta/why-interesting.jsx';
+import IssuesAndTags from './meta/issues-and-tags.jsx';
+import BookmarkControl from '../bookmark-control.jsx';
+import bookmarkManager from '../../js/bookmarks-manager';
+import user from '../../js/app-user.js';
+
+class ProjectCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bookmarked: false
+    };
+  }
+
+  sendGaEvent(category = ``, action = ``, transport = ``) {
+    let config = {
+      category: `Entry Card - ${category}`,
+      action: action,
+      label: `${this.props.id} - ${this.props.title}`
+    };
+
+    if (transport) {
+      config.transport = transport;
+    }
+
+    ReactGA.event(config);
+  }
+
+  componentDidMount() {
+    this.setInitialBookmarkedStatus();
+  }
+
+  setInitialBookmarkedStatus() {
+    if (user.loggedin) {
+      this.setState({
+        bookmarked: this.props.isBookmarked
+      });
+
+      return;
+    }
+
+    let bookmarks = bookmarkManager.bookmarks.get();
+    let bookmarked;
+
+    if (bookmarks) {
+      bookmarked = bookmarks.indexOf(this.props.id) > -1;
+      this.setState({bookmarked: bookmarked});
+    }
+  }
+
+  updateCardBookmarkedState(isBookmarked) {
+    this.setState({ bookmarked: isBookmarked });
+  }
+
+  handleReadMoreClick() {
+    this.sendGaEvent(`Read more`, `Clicked`);
+  }
+
+  renderActionPanel(detailViewLink) {
+    if (this.props.onModerationMode) return null;
+
+    return (
+      <div className="action-panel">
+        <div>
+          <Link to={detailViewLink} onClick={() => this.handleReadMoreClick() }>Read more</Link>
+        </div>
+        <BookmarkControl id={this.props.id}
+                         title={this.props.title}
+                         isBookmarked={this.props.isBookmarked}
+                         updateCardBookmarkedState={(bookmarked) => { this.updateCardBookmarkedState(bookmarked); }} />
+      </div>
+    );
+  }
+
+  renderModerationPanel() {
+    if (!this.props.onModerationMode) return null;
+
+    return <ModerationPanel id={this.props.id} moderationState={this.props.moderationState} />;
+  }
+
+  renderExtraMeta() {
+    if (!this.props.onModerationMode) return null;
+
+    return <div>
```

Comment:
This feels like a repetition of `project-card-detailed.jsx`. The same "there are a suspicious number of imports" is the case here, too.